src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java

```diff
@@ -51,23 +53,31 @@ public CaseDocument bundle(List<DocumentMetaData> documents, String authorisatio
         Optional<Document> stitchedDocument = caseDataFromBundlePayload.getCaseBundles().get(0).getValue().getStitchedDocument();
 
         log.info("stitchedDocument.isPresent() {}, legacy case reference {}",  stitchedDocument.isPresent(), caseData.getLegacyCaseReference());
-        return retrieveCaseDocument(stitchedDocument);
+        return retrieveCaseDocument(stitchedDocument, caseData);
 
     }
 
-    private CaseDocument retrieveCaseDocument(Optional<Document> stitchedDocument) {
+    private CaseDocument retrieveCaseDocument(Optional<Document> stitchedDocument, CaseData caseData) {
         if (stitchedDocument.isEmpty()) {
             log.info("stitchedDocument is not present----------");
             return null;
         }
         Document document = stitchedDocument.get();
         String documentUrl = document.getDocumentUrl();
         String documentBinaryUrl = document.getDocumentBinaryUrl();
+
+        LocalDateTime responseDate = null;
+        if (Objects.nonNull(caseData.getRespondent1ResponseDate())) {
+            responseDate = caseData.getRespondent1ResponseDate();
+        } else if (Objects.nonNull(caseData.getRespondent2ResponseDate())) {
+            responseDate = caseData.getRespondent2ResponseDate();
+        }
+
         return CaseDocument.builder()
             .documentLink(Document.builder().documentUrl(documentUrl).documentBinaryUrl(documentBinaryUrl).documentFileName(document.getDocumentFileName()).build())
             .documentName("Stitched document")
             .documentType(SEALED_CLAIM)
-            .createdDatetime(LocalDateTime.now())
+            .createdDatetime(caseData.getCreatedResponseDate(responseDate))
```

Comment:
its the caseData that has to return the date without us having to retrieve the parameter. the knowledge is already in casedata

Reply:
done